src/zino/scheduler.py

```diff
@@ -93,8 +94,8 @@ def schedule_devices(devices: Sequence[str]):
 
     scheduler = get_scheduler()
 
-    # Spread poll jobs evenly across the entire default interval
-    stagger_factor = (DEFAULT_INTERVAL_MINUTES * 60) / len(devices)
+    # Spread poll jobs evenly across the entire interval
+    stagger_factor = (stagger_interval * 60) / len(devices)
```

Comment:
for me doing this convertion here feels not logical. Isn't it better to move this outside the schedule_devices function and up to load_and_schedule_polldevs? That way you just input the number of seconds between devices to the schedule_devices function and keep logic outside it.

Reply:
This was not logic introduced in this pull request, so in order to fix this bug I would suggest creating a new issue where we can discuss where this logic best belongs. 
Because to me it feels at the right place. The function `load_and_schedule_polldevs` should not have to handle details on how to actually schedule the jobs, only that they are being scheduled